torchsynth/nebulae/voice/default.json

```diff
@@ -0,0 +1,1250 @@
+   "lfo_2",
+   "initial_phase",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2",
+   "sin",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2",
+   "sin",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2",
+   "tri",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2",
+   "tri",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2",
+   "saw",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2",
+   "saw",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2",
+   "rsaw",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2",
+   "rsaw",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2",
+   "sqr",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2",
+   "sqr",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "attack",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "attack",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "decay",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "decay",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "sustain",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "sustain",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "release",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "release",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "alpha",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2_amp_adsr",
+   "alpha",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "attack",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "attack",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "decay",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "decay",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "sustain",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "sustain",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "release",
+   "curve"
+  ],
+  "value": 0.5
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "release",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "alpha",
+   "curve"
+  ],
+  "value": 1
+ },
+ {
+  "name": [
+   "lfo_2_rate_adsr",
+   "alpha",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
+   "mixer",
+   "level0",
+   "curve"
+  ],
+  "value": 1.0
+ },
+ {
+  "name": [
+   "mixer",
+   "level0",
+   "symmetric"
+  ],
+  "value": false
+ },
+ {
+  "name": [
```

Comment:
just thinking forward to potentially having nested modules -- in which case it would be easier to keep these as just lists. If we have a nested module we just add another value. for example we could have `["fm_operator_1", "vco", "tuning", "curve"] and it would work with this format

Reply:
okay

Reply:
I don't think this will be the right approach for nesting but we can keep it for now